vnext/PropertySheets/ReactCommunity.cpp.props

```diff
@@ -15,13 +15,7 @@
     </ClCompile>
   </ItemDefinitionGroup>
 
-  <ItemDefinitionGroup Condition="'$(OSS_RN)'=='true'">
-    <ClCompile>
-      <PreprocessorDefinitions>OSS_RN;ENUM_BITFIELDS_NOT_SUPPORTED;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
ENUM_BITFIELDS_NOT_SUPPORTED is not checked in facebook/react-native or folly. Safe to remove this, since it seems historical